src/components/calcite-scrim/calcite-scrim.tsx

```diff
@@ -0,0 +1,52 @@
+import { Component, Host, Prop, h } from "@stencil/core";
+import { CSS } from "./resources";
+
+@Component({
+  tag: "calcite-scrim",
+  styleUrl: "calcite-scrim.scss",
+  shadow: true
+})
+export class CalciteScrim {
+  // --------------------------------------------------------------------------
+  //
+  //  Properties
+  //
+  // --------------------------------------------------------------------------
+
+  /**
+   * When true, scrim adds the aria-disabled attribute
```

Comment:
These properties are a little odd because they mention aria.

Reply:
I could use some advice on this.
My thought was that if the user wants to handle `aria-busy` or `aria-disabled` on their own, they could turn the scrim's attributes off. But it'd be nice to have the scrim come with these by default.

Reply:
@driskull Actually, I am gonna remove all the aria busy and disabled stuff and leave it to the user to place those attributes.

Reply:
ok sounds good.